docker/nginx/nginx-key.conf

```diff
@@ -125,6 +125,23 @@ http {
             include snippets/ws-upgrade.conf;
             proxy_pass http://ws-adapter:${SMQ_WS_ADAPTER_HTTP_PORT}/;
         }
+
+        # Proxy Pass to the Magistrala UI
+        location /ui/ {
```

Comment:
In Docker Compose file currently doesn't include ui or ui-backend, so Nginx won't start properly until all the endpoints are accessible. To make this PR functional, we need to add ui and ui-backend to the Docker Compose configuration. Without these, this PR won't be effective.

Reply:
We will need to add them, as well. Similar to what we did for Magistrala. @Musilah Please take care of this.